chain/chain/src/resharding/manager.rs

```diff
@@ -221,7 +217,7 @@ impl ReshardingManager {
             let mem_trie_update = mem_tries.update(*parent_chunk_extra.state_root(), mode)?;
 
             let trie_changes = mem_trie_update.retain_split_shard(&boundary_account, retain_mode);
-            let mem_changes = trie_changes.mem_trie_changes.as_ref().unwrap();
+            let mem_changes = trie_changes.memtrie_changes.as_ref().unwrap();
```

Comment:
+1 and it would also be nice to fix the other inconsistencies in naming - e.g. mem_changes -> memtrie_changes here, mem_trie_changes -> memtrie_changes below, mem_trie_update above

no need to do it in this PR though

Reply:
Let me do most of these as a separate PR!

Reply:
btw personally I prefer mem_trie and MemTrie over memtrie and Memtrie but consistency above all ;)

Reply:
Lol! My deep dark secret is that I prefer `memtrie` and `MemTrie` but these two don't really fit together 😂